sharness/t0050-service-state-upgrade-from-current.sh

```diff
@@ -0,0 +1,24 @@
+#! /bin/sh
+
+test_description="Test service state 'upgrade' from current version"
+
+. lib/test-lib.sh
+
+test_ipfs_init
+cleanup test_clean_ipfs
+test_cluster_init
+cleanup test_clean_cluster
+
+test_expect_success IPFS,CLUSTER "cluster-service state preserved by migration" '
+    cid=`docker exec ipfs sh -c "echo testing | ipfs add -q"` &&
```

Comment:
These 3 first don't need to be &&-chained I guess, since whats below will fail if they didn't work